exporter/templates_test.go

```diff
@@ -264,7 +264,6 @@ func TestDeviceAppliedTemplate(t *testing.T) {
 	xrn := xmlRootNodeForQrdaOidWithData(qrdaOid, ei)
 
 	assertXPath(t, xrn, "//entry/procedure", map[string]string{"classCode": "PROC", "moodCode": "EVN"}, nil)
-	assertXPath(t, xrn, "//entry/procedure/templateId[@root='2.16.840.1.113883.10.20.22.4.14']", nil, nil)
```

Comment:
Good eye. This isn't related to the diagnostic study templates, but I noticed this when writing my tests. The deleted lines assert the oid of the parent templates. I did that because I didn't think the parent necessarily had to be that template, but I could definitely be wrong about that.  @okeefm ?